yew-router/src/switch.rs

```diff
@@ -126,11 +126,8 @@ impl<U: Switch> Switch for Permissive<U> {
     }
 }
 
-// TODO the AllowMissing shim doesn't appear to offer much over Permissive.
-// Documentation should improve (need examples - to show the difference) or it should be removed.
-
 /// Allows a section to match, providing a None value,
-/// if its contents are entirely missing, or starts with a '/'.
+/// if its contents are entirely missing, or ends with a '/' without any.
```

Comment:
I'm not exactly sure what `without any` refers to.

Reply:
it's the contents without any.

Reply:
That's no problem!
I just want to make it clear for people who stumble across this in the future :)
Do you mean that there's nothing after the slash (i.e. `/` but not `/<something here>`

Reply:
Yes it is.

Reply:
```suggestion
/// if it doesn't have any contents, or ends with a slash ('/') with no characters after it.
```

Reply:
Thanks, it's more readable now.
Probably `matches` would be better than `characters` because there's a case like `/#` which also ends as `AllowMissing(None)`.

Reply:
Or `matched characters` would be better?

Reply:
Perhaps mentioning which characters don't count?

Reply:
Well, cases like `#a=1` and `?b=1` would also be ignore. There are just too many cases that could be ignored.
Besides, this chracter matching is what `matcher` part of code does.
Too much information on another file would make people confused.

Reply:
That makes sense.